Gemfile

```diff
@@ -1,18 +1,18 @@
 source 'https://rubygems.org'
 
-gem 'rails', '4.2.6'
+gem 'rails', '~> 5.0.1'
 gem 'pg', '~> 0.18.4'
 gem 'sass-rails', '~> 5.0'
 gem 'uglifier', '>= 1.3.0'
 gem 'active_model_serializers'
 gem 'responders'
 gem 'rack-haproxy_status', '~> 0.8.1'
 
-gem 'sprockets-rails', '~> 3.0.4'
+gem 'sprockets-rails'
```

Comment:
Ja. Zelf zie ik de reden er niet direct van in, versies worden toch al in het Gemfile.lock bestand bijgehouden. Bij elke gem update zal dan de versie weer handmatig aangepast moeten worden. Voor zover ik weet is gemnasium ook niet optimaal te gebruiken met harde versies.